### PR TITLE
Add Input documentation for `exact_match`

### PR DESCRIPTION
**Pull request by raulsntos**
Adds documentation for PR #44355

### CODE REVIEW

File: doc/classes/Input.xml [hunk 1]
```diff
@@ -60,6 +60,7 @@
 			<argument index="0" name="action" type="StringName">
 			</argument>
 			<argument index="1" name="exact_match" type="bool" default="false">
+				If [code]true[/code] the event that contributes to the raw strength only matches the action if the event provided exactly matches the action event. For an action like [code]KEY_S[/code], the event [code]KEY_CONTROL + KEY_S[/code] matches if [code]exact_match[/code] is [code]false[/code] but not if it is [code]true[/code].
```
Comment (akien-mga) [inline doc/classes/Input.xml:63]: You can't add documentation for arguments this way, it should be part of the `<description>`.

Comment (raulsntos) [reply doc/classes/Input.xml:63]: So should I move the text to inside the `<description>` tag and leave the `<argument>` tag empty?

Comment (akien-mga) [reply doc/classes/Input.xml:63]: Yes, `argument` tags don't support text.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**